### ISSUE TITLE
logevent.py duration function fails when log line includes WTCheckpointThread

### ISSUE THREAD

**Issue by niccottrell** [rueckstiess/mtools#762]
### Expected behavior
Log lines like the following should not crash duraiton: 
```
2019-12-26T14:00:19.539+0000 I  STORAGE  [WTCheckpointThread] WiredTiger message read timestamp (1577368819,3971) less than the oldest timestamp (1577368819,4078) 
```

### Steps to reproduce the actual/current behavior
mplotqueries fails with:
```
all last):
  File "/Users/nic/anaconda3/bin/mplotqueries", line 10, in <module>
    sys.exit(main())
  File "/Users/nic/anaconda3/lib/python3.7/site-packages/mtools/mplotqueries/mplotqueries.py", line 714, in main
    tool.run()
  File "/Users/nic/anaconda3/lib/python3.7/site-packages/mtools/mplotqueries/mplotqueries.py", line 146, in run
    self.parse_logevents()
  File "/Users/nic/anaconda3/lib/python3.7/site-packages/mtools/mplotqueries/mplotqueries.py", line 256, in parse_logevents
    if self.plot_instance.accept_line(logevent):
  File "/Users/nic/anaconda3/lib/python3.7/site-packages/mtools/mplotqueries/plottypes/scatter_type.py", line 59, in accept_line
    return getattr(logevent, self.field) is not None
  File "/Users/nic/anaconda3/lib/python3.7/site-packages/mtools/util/logevent.py", line 222, in duration
    self._duration = int(groups.group(1)) * 1000
AttributeError: 'NoneType' object has no attribute 'group'
```

### Environment
| Software         | Version
| ---------------- | -------
| mtools           |  1.6.0

**Comment by niccottrell** [comment]
I believe https://github.com/rueckstiess/mtools/blob/develop/mtools/util/logevent.py#L218 should be
```
 elif "Checkpoint" in self.line_str:
                matchobj = re.search("Checkpoint took ([\d]+) seconds to complete", self.line_str)
                if matchobj is not None:
                    self._duration = int(matchobj.group(1)) * 1000
``

**Comment by sommeric** [comment]
I recommend changing to:

elif "Checkpoint took" in self.line_str:

to exclude the undesired line, even if the "not None" line is added to catch anything else that's unexpected.

**Comment by niccottrell** [comment]
In fact every instance of `if X: ` on a result from `re.search(...)`should probably be changed to `if X is not None:`

**Comment by stennie** [comment]
Dupe of #757; fix will be included in 1.6.1

**Comment by stennie** [comment]
@sommeric 
> I recommend changing to:
>
> elif "Checkpoint took" in self.line_str:
>
>to exclude the undesired line, even if the "not None" line is also added to catch anything else that's unexpected.

Good suggestion, thanks!
